mnist/README.md

```diff
@@ -528,14 +547,13 @@ The number `1547100373` is a version number auto-generated by TensorFlow; it wil
 Set your model path
 
 ```
-ks param set ${ENV} mnist-deploy-gcp modelBasePath ${EXPORT_DIR}
-
+ks param set --env=${KSENV} mnist-deploy-gcp modelBasePath ${EXPORT_DIR}
```

Comment:
--env=${KSENV} is optional

Reply:
every other line in the tutorial used --env=${KSENV}, so I wanted to make sure this was consistent. Do you think we should mention it's optional somewhere?

Reply:
I see, then that's fine